understanding/20/images-of-text.html

```diff
@@ -44,6 +44,10 @@ <h2>Intent of Images of Text</h2>
       <p>Images of text can also be used where it is possible for users to customize the image
          of text to match their requirements.
       </p>
+
+      <p>If it is not possible to replace an image of text with actual text or to allow users to customize the image, the author must provide an [accessible alternative](https://www.w3.org/TR/WCAG22/#cc2) by duplicating the meaning or content of the image in text form. 
```

Comment:
It is unclear what you are trying to link to with this URL: https://www.w3.org/TR/WCAG22/#cc2

There is no definition of _accessible alternative_. Are you making the case that the page can conform although it fails 1.4.5, by meeting 1.1.1?

Reply:
> It is unclear what you are trying to link to with this URL: https://www.w3.org/TR/WCAG22/#cc2

I'm referring to NOTE 1, which states:

> For the purpose of determining conformance, alternatives to part of a page's content are considered part of the page when the alternatives can be obtained directly from the page, e.g., a long description or an alternative presentation of a video.

----

> Are you making the case that the page can conform although it fails 1.4.5, by meeting 1.1.1?

Not really. SC 1.4.5 and 1.1.1 address different needs. SC 1.1.1 is primarily intended for people who can rely on text alternatives that don't need to be visible, while SC 1.4.5 requires an option to visually customize the text. Therefore, if the same text is presented close to the image, it addresses the requirement.